src/QMCWaveFunctions/lcao/LCAOrbitalSet.h

```diff
@@ -48,6 +50,8 @@ namespace qmcplusplus
     ///Tempv(OrbitalSetSize) Tempv=C*Temp
     vgl_type Tempv; 
 
+    CuspCorrectionBase *cusp;
```

Comment:
I don't feel right with this addition and all the ifs in the cpp. It is better to have LCAOrbitalSetWithCuspCorrection as a derived class of LCAOrbitalSet. In the derived class, call LCAOrbitalSet::evaluate() + cusp evaluation.